client/src/components/MonthSelector.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import FormControl from '@material-ui/core/FormControl';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from "@material-ui/core/Select";
+
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(theme => ({
+  formControl: {
+    position: 'absolute',
+    top: props => `${props.top}`,
+    right: props => `${props.right}`,
+    margin: theme.spacing(1),
+    minWidth: 120,
+  }
+}));
+
+export function MonthSelector(props) {
+  const { top, right } = props;
+  const classes = useStyles({ top, right });
```

Comment:
i don't think these classes are being used in this component so it should not be passed down from props and created in class

Reply:
the `top` and `right` are assigned in the props of `MonthSelector`. You can see the assigned values in `<Dashboard />` and `<Receipts />`